cucumber-core/src/test/java/io/cucumber/core/plugin/MessageFormatterTest.java

```diff
@@ -32,9 +39,21 @@ void test() {
         bus.send(Envelope.of(testRunFinished));
 
         String ndjson = new String(bytes.toByteArray(), UTF_8);
-        assertThat(ndjson, containsString("" +
-                "{\"testRunStarted\":{\"timestamp\":{\"seconds\":10,\"nanos\":0}}}\n" +
-                "{\"testRunFinished\":{\"success\":true,\"timestamp\":{\"seconds\":15,\"nanos\":0}}}\n"));
+        String expectedJson = "{\"testRunStarted\":{\"timestamp\":{\"seconds\":10,\"nanos\":0}},\"testRunFinished\":{\"success\":true,\"timestamp\":{\"seconds\":15,\"nanos\":0}}}";
```

Comment:
This meaningfully changes what is being tested. Json and Ndjson are distinctly different. See https://ndjson.org/